ajax/libs/eldarion-ajax/package.json

```diff
@@ -1,16 +1,27 @@
 {
   "name": "eldarion-ajax",
-  "filename": "eldarion-ajax.min.js",
-  "version": "0.12.0",
+  "filename": "js/eldarion-ajax.min.js",
+  "version": "0.16.0",
   "description": "a library for adding declarative ajax functionality to your website",
-  "homepage": "https://github.com/eldarion/eldarion-ajax/",
   "keywords": [
     "ajax"
   ],
   "repository": {
     "type": "git",
     "url": "git://github.com/eldarion/eldarion-ajax.git"
   },
-  "license": "BSD",
-  "author": "Patrick Altman"
+  "license": "BSD-3-Clause",
+  "author": "Patrick Altman",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/eldarion/eldarion-ajax.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/eldarion-ajax.min.js"
```

Comment:
How about using `js/+(eldarion-ajax|polyfills)*` to include source files.
See https://github.com/eldarion/eldarion-ajax/blob/0.16.0/build.sh.